contracts/ForkDelta.sol

```diff
@@ -115,31 +63,31 @@ contract ForkDelta is SafeMath {
   }
 
   function deposit() public payable {
-    tokens[0][msg.sender] = safeAdd(tokens[0][msg.sender], msg.value);
-    Deposit(0, msg.sender, msg.value, tokens[0][msg.sender]);
+    tokens[0][msg.sender] = tokens[0][msg.sender].add(msg.value);
+    emit Deposit(0, msg.sender, msg.value, tokens[0][msg.sender]);
   }
 
   function withdraw(uint amount) public {
     require(tokens[0][msg.sender] >= amount);
-    tokens[0][msg.sender] = safeSub(tokens[0][msg.sender], amount);
-    require(msg.sender.call.value(amount)());
-    Withdraw(0, msg.sender, amount, tokens[0][msg.sender]);
+    tokens[0][msg.sender] = tokens[0][msg.sender].sub(amount);
+    msg.sender.transfer(amount);
```

Comment:
Why did we remove the require here?
Why did we change `call.value` to `.transfer` throughout?

Reply:
Previously: require throws when function returns false, e.g. when not enough eth.
New version: transfer will not return a result, but throw on fail itself, so no require needed

You could also say that transfer is safer to use, because you don't have to check the result.

Reason for the change: Safety against reentrancy as explained in:

https://consensys.github.io/smart-contract-best-practices/recommendations/#be-aware-of-the-tradeoffs-between-send-transfer-and-callvalue

Reply:
Beautiful.